active/Expander/Expander.md

```diff
@@ -0,0 +1,173 @@
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+ > This spec corresponds to [this issue](https://github.com/microsoft/microsoft-ui-xaml/issues/3279) on the WinUI repo. 
+
+Throughout Windows, different expander controls are used by various apps and surfaces. There’s currently no consistent way to address this common UX pattern. This control is needed for situations where expanding (instead of overlaying) content is needed.  An Expander control is motivated by its use in many app scenarios and supporting developers in migrating from [WPF](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/expander-overview?view=netframeworkdesktop-4.8) and [WCT](https://docs.microsoft.com/en-us/windows/communitytoolkit/controls/expander).  
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+An Expander control is a UI component that provides a standard interaction for showing more content in a container that pushes adjacent content while expanding and collapsing. An Expander cannot be light dismissed and is independent of the contents inside it, including controls. Expander should be used when some content is only relevant some of the time (for example to read more information or access additional options for an item). 
+
+![A collapsed Expander](images/expander_collapsed.PNG)
+![An expanded Expander](images/expander_expanded.PNG)
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+## Create an Expander
+
+![A collapsed Expander](images/expander_collapsed.PNG)
+XAML
+~~~~
+<muxc:Expander x:Name="SimpleExpander" />
+~~~~
+## Put controls inside an Expander
+![An expanded Expander with a ToggleButton in the Header and a Button in the Content](images/expander_example_buttons.PNG)
+XAML
+~~~~
+<muxc:Expander x:Name="Expander2" IsExpanded="False" ExpandDirection ="Down">
+    <muxc:Expander.Header>
+        <ToggleButton>This is a ToggleButton in the header</ToggleButton>
```

Comment:
Do I have to put the ToggleButton in the Header myself? I was expecting I could put any content in there (typically a string), and the toggling behavior would be automatic.

Reply:
That is correct, you should not have to put in a toggle button.

Reply:
I'm updating the examples to clarify this - sorry for the confusion! The intention was to show that you can put another control (like a ToggleButton) inside the Header.